docs/production-deployment/cloud/terraform-provider.mdx

```diff
@@ -24,6 +24,12 @@ Temporal supports the [Cloud Namespace](/cloud/namespaces) resource management,
 
 :::
 
+:::info
+
+GCP (Google Cloud Platform) is not supported by the Temporal Cloud Terraform provider.
```

Comment:
Should we add a `for now` or something here?
i.e. the intent is to support it but it's not yet there

Reply:
From my understanding we don't speculate on future availability in docs (even though we personally know it's coming, we avoid referring to future releases as it dates the docs.) right @fairlydurable

Reply:
To the best of my understanding, wrt Docs team guidance: we are happy to update docs when a feature ships. We do not promise future features as plans often change and we do not want to commit Temporal as an organization to any specifics.

Reply:
Sounds good. @XaF Are we certain GCP isn't supported by the provider yet?

Reply:
Some of the cloud ops APIs exercised by the terraform provider are not yet ported to GCP, so yeah not supported at the moment :( (cc @jumbogo)

Reply:
Sounds good. Thanks for the verification